docs/usecase.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+left to right direction
+skinparam packageStyle rectangle
+actor customer
+actor receptionist
+rectangle "Camping reservation system" {
+  customer <|-- (Generate invoice based on reservation details)
+  customer -- (Search availability)
+  customer -- (Book a camping)
+  (Book a camping) .> (Electricity) : <<include>>
+  (Manage reservations) -- receptionist
+  (Manage reservations) .> (Cancel reservation) : <<include>>
+  (Manage reservations) -up.> (Book a camping) : <<include>>
+  customer -- (Cancel reservation)
+  (See campsite layout) -- receptionist
+}
+@enduml
```

Comment:
A távozáshoz is kellene egy use case és szerintem szükség van a recepciósokat azonosítani, ezért egy login use case is kell.